cmd/minibroker/main.go

```diff
@@ -154,7 +154,7 @@ func runWithContext(ctx context.Context) error {
 }
 
 func cancelOnInterrupt(ctx context.Context, f context.CancelFunc) {
-	term := make(chan os.Signal)
+	term := make(chan os.Signal, 1)
```

Comment:
Thought `make(chan …)` defaulted to 1. Doesn't hurt either way, at least.

Reply:
Not true. The former was an unbuffered channel. I believe this change was `go vet` yelling at me.